pandas/core/strings.py

```diff
@@ -166,7 +189,7 @@ def str_contains(arr, pat, case=True, flags=0, na=np.nan, regex=True):
 
     Returns
     -------
-    Series of boolean values
+    contained : Series/array (bool)
```

Comment:
I find "boolean Series" or "Series of boolean values" actually a bit more clear as bool between brackets